full/src/main/java/apoc/mongodb/MongoDBUtils.java

```diff
@@ -76,7 +74,7 @@ protected static MongoDBUtils.Coll getMongoColl(Supplier<Coll> action) {
                     "\n" +
                     "mongodb-driver-core-x.y.z.jar\n" +
                     "\n" +
-                    "jackson-annotations-x.y.z.jar\n\njackson-core-x.y.z.jar\n\njackson-databind-x.y.z.jar\n\nSee the documentation: https://neo4j.com/labs/apoc/%s/database-integration/mongodb/", version));
+                    "jackson-annotations-x.y.z.jar\n\njackson-core-x.y.z.jar\n\njackson-databind-x.y.z.jar\n\nSee the documentation: https://neo4j.com/labs/apoc/4.4/database-integration/mongodb/"));
```

Comment:
There's no alternative to this at the moment because this lives in full and `Version` comes from `core`. We should have an address `/current` like the neo4j docs have